src/cljs/rems/spa.cljs

```diff
@@ -362,6 +354,10 @@
   (rf/dispatch [:rems.administration.create-workflow/enter-page])
   (rf/dispatch [:set-active-page :rems.administration/create-workflow]))
 
+(secretary/defroute "/extra-pages/:page-id" [page-id]
```

Comment:
we might need to change this later if some user doesn't like the `/extra-pages/` part in the url, but let's worry about it then

Reply:
Ok, I didn't do anything about it now.